### DISCUSSION TITLE
Are Preview deployments less performant?

### DISCUSSION THREAD

**Discussion by jacksondoherty** [vercel/community#2142] [Help]
### Summary

I'm thinking about using preview deployments that different versions of my iOS app point to. Is this a good idea / are preview deployments as performant as production?

**Comment by amyegan** [comment]
I would not recommend using preview deployments for production purposes. Instead, you can create a separate project to deploy a different version

**Reply by jacksondoherty** [reply]
Any specifics as to why? It currently makes for a very convenient workflow with git branches, etc.!
